deploy/ftlib-test.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ftlib-test
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: ftlib-test-app
+  template:
+    metadata:
+      labels:
+        app: ftlib-test-app
+    spec:
+      containers:
+        - image: ftlib:deploy
+          imagePullPolicy: Never
+          name: ftlib-test-c
+          ports:
+          - containerPort: 3000
+            name: http
+          - containerPort: 22
+            name: ssh
+          command: ["python3", "/ftlib/test/generic_test.py"]
+          resources:
```

Comment:
The `dummy_NCCL` needs GPUs for communication, which seems not budget-friendly.

Reply:
Travis doesn't have GPUs for testing. I updated this command to run pytorch test instead.